### PR TITLE
Practice2 pull request

### PR DESCRIPTION
**Pull request by andrew-budnitskiy**
This is pull request for the **second** practice.
Created by Andrew Budnitskiy.
28.06.2021

### CODE REVIEW

File: Cats/Cats/ViewController.swift [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+        case 1:
+            return 195
+        default:
+            return 350//tableView.bounds.size.width / 2.0
+
+        }
+
+    }
+    
+    //В главной таблице 3 ячейки
```
Comment (vasilevkin) [inline Cats/Cats/ViewController.swift:39]: table should contain all items in the animals list

**Review by vasilevkin — CHANGES_REQUESTED** [review]

**Comment by vasilevkin** [comment]
it looks better now.

Please add SwiftLint to the project and proceed with additional tasks or Practice 3.

File: Cats/Cats/AppDelegate.swift [hunk 2]
```diff
@@ -0,0 +1,21 @@
+//
```
[the remark this answers is not in the document]
Comment (andrew-budnitskiy) [reply Cats/Cats/AppDelegate.swift]: Issued

File: Cats/Cats/Info.plist [hunk 3]
```diff
@@ -0,0 +1,47 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
[the remark this answers is not in the document]
Comment (andrew-budnitskiy) [reply Cats/Cats/Info.plist]: Issued

File: Cats/Cats/ViewController.swift (hunk 1 above)
Comment (andrew-budnitskiy) [reply Cats/Cats/ViewController.swift:133]: Some transformations were made, and this comment was issued too.

**Comment by andrew-budnitskiy** [comment]
> it looks better now.
> 
> Please add SwiftLint to the project and proceed with additional tasks or Practice 3.

I'm sorry, but there is'nt additional tasks in this practice - there are in Practice1 or Practice3. Is it necessary add SwiftLint to this practice exactly?

**Comment by andrew-budnitskiy** [comment]
> it looks better now.
> 
> Please add SwiftLint to the project and proceed with additional tasks or Practice 3.

Issued

**Comment by vasilevkin** [comment]
proceed with Practice 3.

**Review by alekseykorol — APPROVED** [review]

**Pull request merged**